applications/crossbar/priv/couchdb/schemas/auth_provider.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "_id": "auth_provider",
+    "additionalProperties": true,
+    "description": "authentication providers",
+    "properties": {
+        "enabled": {
+            "default": true,
+            "description": "whether or not this authentication provider is enabled",
+            "type": "boolean"
+        },
+        "provider_name": {
+            "description": "authentication provider name",
+            "required": true,
+            "type": "string"
+        },
+        "provider_type": {
+            "description": "the type of authentication provider",
+            "enum": [
+                "multi_factor",
+                "oauth"
+            ],
+            "required": true,
+            "type": "string"
+        },
+        "settings": {
+            "description": "provider configurations",
+            "type": "object"
+        }
+    },
+    "required": [
+        "provider_name",
+        "provider_type"
+    ],
```

Comment:
Why they're not required? 
`provider_type` is required, otherwise view is not working properly,
and `provider_name` is required too because `kz_mfa_auth` needs it.

`enabled` is `true` by default though.